pkg/services/ngalert/provisioning/contactpoints.go

```diff
@@ -118,7 +118,7 @@ func (ecp *ContactPointService) getContactPointDecrypted(ctx context.Context, or
 		}
 		embeddedContactPoint, err := PostableGrafanaReceiverToEmbeddedContactPoint(
 			receiver,
-			models.ProvenanceNone,
+			models.ProvenanceNone, // TODO should be correct provenance?
```

Comment:
It doesn't matter currently; this is only used during the Update flow to fetch the decrypted secrets, this value is thrown away. There is a separate, dedicated query and provenance check later on.